BasicAnalysis/README.md

```diff
@@ -0,0 +1,99 @@
+## Integration Guide
+
+This guide offers comprehensive instructions for seamlessly integrating KKSPlayerCollector into your
```

Comment:
This guide shows how to send player logs to the player's default analysis service by setting the AnalyticsConfig when creating the player.